css-four-column-layout/styles.css

```diff
@@ -4,47 +4,46 @@
 
 body {
   margin: 0;
+  font-family: Barlow, sans-serif;
 }
 
 h2 {
-  text-align: center;
-  font-weight: bold;
-  font-family: Barlow, sans-serif;
-  position: relative;
-  left: 30px;
+  font-size: 18px;
 }
 
-h3 {
-  font-weight: bold;
-  font-family: Barlow, sans-serif;
+.container {
+  width: 1000px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.header {
+  text-align: center;
 }
 
-p {
-  padding: 20px;
+.row {
+  display: flex;
+  justify-content: center;
+  text-align: center;
```

Comment:
The `row` class should **_only_** have `display: flex`, anything else should be on a separate class that you can add to the element in addition to the `row` class. So for instance:
```
<div class="row justify-center text-center">
```